integration_tests/mockApis/assessments.ts

```diff
@@ -29,4 +29,16 @@ export default {
         jsonBody: assessment,
       },
     }),
+  stubAssessmentUpdate: (args: { assessment: ApprovedPremisesAssessment }): SuperAgentRequest =>
+    stubFor({
+      request: {
+        method: 'PUT',
+        url: `/assessments/${args.assessment.id}`,
```

Comment:
Would be great if this changed later to use the paths setup in the next commit (though I recognised that would be inconsistent with elsewhere)